frontend/packages/ceph-storage-plugin/src/plugin.tsx

```diff
@@ -0,0 +1,27 @@
+import * as _ from 'lodash-es';
```

Comment:
FYI, I wrote a couple of PRs to eliminate `lodash` vs `lodash-es` duality: #1708, #1724.

(No need to change anything here for now.)

Reply:
Ack! Will keep an eye on these PR's once merged will update them to use `lodash`. Thanks